src/commands/transaction/pullfunds.ts

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
This command was removed

Reply:
Did you mean delete this command `pullfunds` ?

Reply:
Yes, the command  ``pullfund`` was removed in a previous PR.